src/navmenu/NavMenu.css

```diff
@@ -7,6 +7,11 @@
   font-weight: 400;
 }
 
+.MuiListItem-root.nav-menu-item {
```

Comment:
This changes are from a different branch. They are here since I didn't branch out from master.

Reply:
Can we revert this change since it's being handled by your other branch? Just to keep this focused on these changes.

Reply:
Since deletion would be a later commit, these changes will be gone when we merge. (Since they are already in master). I will make sure to branch out of master next time.